packages/firestore/test/integration/api/transactions.test.ts

```diff
@@ -809,18 +836,18 @@ apiDescribe('Database transactions', (persistence: boolean) => {
   it('are cancelled on rejected promise', () => {
     return integrationHelpers.withTestDb(persistence, db => {
       const doc = db.collection('towns').doc();
-      let count = 0;
+      let counter = 0;
```

Comment:
I'm not opposed to renaming `count` to `counter`, but you just added a new test above that uses `count`...  so we're not being particularly consistent here... :)